src/avram/test.cr

```diff
@@ -0,0 +1,45 @@
+module Avram::Test
```

Comment:
Is this the official name, or just for testing purposes?

Reply:
I didn't put much consideration into this name. What do you think would be better?

Reply:
hmm... `Avram::SpecTransaction`... `Avram::TransactionSpecHelper` ? 🤔  I just thought `Test` was you trying things out and you were gonna rip this out when you were done. I wasn't sure if it was actual code or not 😅